test/metrics_test.py

```diff
@@ -63,7 +63,11 @@ def test_timer(self, time):
     @mock.patch('shumway.time.time')
     def test_timer_tags(self, time):
         time.side_effect = [0, 1]
-        timer = shumway.Timer('timer', 'key', {'test': 'test'}, ['test'])
+        timer = shumway.Timer('timer',
+                              'key',
+                              {'test': 'test'},
+                              {'res1': 'test'},
```

Comment:
Was the resource required for this? An assertion doesn't seem to change for this addition.